src/server.rs

```diff
@@ -227,6 +235,15 @@ fn read<'r>(
     // against FxA and the method, and either returns OK or an error. We need to reraise it to the
     // handler.
     let request_id = headers.headers.get("request_id");
+
+    let mut tags = Tags::default();
```

Comment:
In here and in `write`: We don't want any of these values as tags (except maybe status?) -- we'll have an arbitrarily high amount of different ids, it's too much cardinality for tags.

Reply:
I thought that [`extra`](https://docs.sentry.io/platforms/javascript/enriching-events/context/) fields weren't treated like normal `tags`, and there wasn't a concern about cardinality? As it is, we're only really adding `status` as a tag, and that state is fairly limited.

Reply:
You're right, sorry I misread this as `tags.tags`